README.md

```diff
@@ -54,22 +43,33 @@ Generate the CSS files
 npm run build
 ```
 
-In order to update an existing Git submodule, you need to execute the “git submodule update” with the “–remote” and the “–merge” option.
+Import the CSS in your own CSS files by using our theme class in your Master HTML template
 
-```bash
-git submodule update --remote --merge
+```HTML
+
+<html lang="nl" class="view openinwoner-theme">
 ```
 
-**Using tokens**
+```css
+.openinwoner-theme {
+. . .
+}
+```
 
-If you are only _consuming_ the design tokens, the easiest integration path is adding this
-package in your own project.
+## Developing and using tokens
 
-Then, import the desired build target artifact and run your usual build chain.
+In case you do not just wish to consume, but add completely new tokens or adjust their values,
+follow these steps. Then, import the desired build target artifact and run your usual build chain.
```

Comment:
You say `follow these steps. Then ...`, but it's not clear what "these steps" are.